driver/src/main/scala/api/commands/writeresult.scala

```diff
@@ -32,8 +32,7 @@ sealed trait WriteResult {
 object WriteResult {
   def lastError(result: WriteResult): Option[LastError] = result match {
```

Comment:
I think to really fix the issue with bulk, some specific error handling should be done.
The previous or the new behaviour of `.lastError` is ok for individual write op.